ioctl/newcmd/bc/bcinfo_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package bc
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-proto/golang/iotexapi"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+	"github.com/iotexproject/iotex-core/test/mock/mock_apiserviceclient"
+	"github.com/iotexproject/iotex-core/test/mock/mock_ioctlclient"
+)
+
+// test for bc info command
+func TestNewBCInfoCmd(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	client := mock_ioctlclient.NewMockClient(ctrl)
+	client.EXPECT().SelectTranslation(gomock.Any()).Return("", config.English).Times(3)
+	client.EXPECT().Config().Return(config.ReadConfig).Times(3)
+
+	apiServiceClient := mock_apiserviceclient.NewMockServiceClient(ctrl)
+	client.EXPECT().APIServiceClient(gomock.Any()).Return(apiServiceClient, nil).Times(1)
+
+	chainMetaResponse := &iotexapi.GetChainMetaResponse{ChainMeta: &iotextypes.ChainMeta{}}
+	apiServiceClient.EXPECT().GetChainMeta(gomock.Any(), gomock.Any()).Return(chainMetaResponse, nil).Times(1)
+
+	cmd := NewBCInfoCmd(client)
+	_, err := util.ExecuteCmd(cmd)
+	require.NoError(t, err)
```

Comment:
for unit test: not only the success case, but also all failure cases:
1. client.EXPECT().APIServiceClient(gomock.Any()).Return(nil, some error)
2. apiServiceClient.EXPECT().GetChainMeta(gomock.Any(), gomock.Any()).Return(nil, some error)

Reply:
fixed